rfc-template.md

```diff
@@ -0,0 +1,67 @@
+- Contribution Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- Target Date: (is there a target date of completion?)
+- RFC PR: [linkerd/rfc#0000](https://github.com/linkerd/rfc/pull/0000)
+- Linkerd Issue: [linkerd/linkerd2#0000](https://github.com/linkerd/linkerd2/issues/0000)
+- Reviewers: (Who should review the code deliverable? ex. @olix0r)
+
+# Summary
+
+[summary]: #summary
+
+One paragraph explanation of the proposed contribution.
+
+# Motivation
+
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+# Reference-level explanation
+
+[reference-level-explanation]: #reference-level-explanation
+
+This is the technical portion of the RFC. Explain the design in sufficient detail that:
+
+- Its interaction with other features is clear.
+- It is reasonably clear how the contribution would be implemented.
+- Corner cases are dissected by example.
+- Dependencies on libraries, tools, projects or work that isn't yet complete.
+
```

Comment:
I would like to see a deliverables section, but that could be a sub-heading under design details.